src/devices_models/devices/common/objective_function/market_bid.jl

```diff
@@ -179,6 +179,36 @@ function _get_pwl_cost_expression(
     )
 end
 
+function _get_pwl_cost_expression_decremental(container::OptimizationContainer,
```

Comment:
Can this be implemented in a way that does not duplicate as much code from the incremental version? Maybe refactor both functions to call a helper function with the common code?